bentoml/cli/bento_management.py

```diff
@@ -167,35 +143,37 @@ def get(bento, limit, ascending_order, print_location, labels, output):
         '--order-by', type=click.Choice(['created_at', 'name']), default='created_at',
     )
     @click.option('--ascending-order', is_flag=True)
+    @click.option('--yatai-url', type=click.STRING, help='Yatai server URL.')
     @click.option(
         '-o',
         '--output',
         type=click.Choice(['json', 'yaml', 'table', 'wide']),
         default='table',
     )
-    def list_bentos(limit, offset, labels, order_by, ascending_order, output):
-        yatai_client = get_default_yatai_client()
-        list_bentos_result = yatai_client.repository.list(
+    def list_bentos_command(
+        limit, offset, labels, order_by, ascending_order, yatai_url, output
+    ):
+        yc = get_yatai_client(yatai_url)
+        result = yc.repository.list(
             limit=limit,
             offset=offset,
             labels=labels,
             order_by=order_by,
             ascending_order=ascending_order,
         )
-        if list_bentos_result.status.status_code != yatai_proto.status_pb2.Status.OK:
-            error_code, error_message = status_pb_to_error_code_and_message(
-                list_bentos_result.status
-            )
-            raise CLIException(f'{error_code}:{error_message}')
-
-        _print_bentos_info(list_bentos_result.bentos, output)
+        _print_bentos_info(result, output)
 
-    @cli.command()
+    @cli.command(
+        name='delete',
+        help='Delete bento. To delete multiple bentos provide the name '
+        'version tag separated by "," for example "bentoml delete name:v1,name:v2',
+    )
     @click.argument("bentos", type=click.STRING, callback=parse_bento_tag_list_callback)
+    @click.option('--yatai-url', type=click.STRING, help='Yatai server URL.')
```

Comment:
"YataiService URL" - let's make sure the naming are consistent across our docs, code and tutorial